### PR TITLE
Adding support for secureTextEntry to TextInput

### PR DESCRIPTION
**Pull request by corymsmith**
Fix for #351

### CODE REVIEW

File: Libraries/Components/TextInput/TextInput.ios.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ var RCTTextViewAttributes = merge(ReactIOSViewAttributes.UIView, {
   mostRecentEventCounter: true,
   placeholder: true,
   placeholderTextColor: true,
+  secureTextEntry: true,
```
Comment (Shrugs) [inline Libraries/Components/TextInput/TextInput.ios.js]: Does this not make it default to true instead of false?

Comment (corymsmith) [inline Libraries/Components/TextInput/TextInput.ios.js]: No, the default is false. That just specifies that those attributes are valid

Comment (Shrugs) [inline Libraries/Components/TextInput/TextInput.ios.js]: Ah, whoops, thanks! 👍

**Comment by mkonicek** [comment]
This has been done in #265.

**Pull request closed without merging**